rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -186,6 +186,8 @@ def acl_del(self, lswitch):
             params = [lswitch]
             self.run("acl-del", args=params)
 
+        # TODO: Don't try to process free-form output from `ovn-nbctl show`.
+        #       Ask for machine readable output, in JSON or CSV, instead.
```

Comment:
Good point. Ideally we can use OVSDB python lib directly, but it might be a little bit overkill. For this ovsclients_impl I think the show command has nothing TODO here. I'd suggest to remove the TODO and create an issue for using OVSDB python lib.

Reply:
Agree, let me drop it and open an issue. For now using `ovn-nbctl` is sufficient for generating the workload but likely it will become a bottleneck as we increase the load. OpenStack Neutron plugin for OVN has started out using `ovn-nbctl` but then had to switch to ovsdbapp lib because the overhead of spawning a process and receiving whole database contents for each operation, like a adding a port, was too costly.